src/_codux/boards/checkout/checkout.board.tsx

```diff
@@ -0,0 +1,19 @@
+import { Checkout } from '../../../components/checkout/checkout';
+import { ContentSlot, createBoard } from '@wixc3/react-board';
+import { ComponentWrapper } from '/src/_codux/board-wrappers/component-wrapper';
+
+export default createBoard({
+    name: 'Checkout',
+    Board: () => (
+        <ComponentWrapper settings={{}}>
+            <ContentSlot>
+                <Checkout />
+            </ContentSlot>
+        </ComponentWrapper>
+    ),
+    isSnippet: true,
+    environmentProps: {
+        canvasMargin: { right: 0, bottom: 0, left: 0, top: 0 },
+        windowWidth: 382,
```

Comment:
can remove this

Reply:
there are the board's props - why it should be removed?